values-nginx.yaml

```diff
@@ -0,0 +1,234 @@
+# Default values for xl-deploy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+## Platform on which to install the chart (PlainK8s/AWSEKS/AzureAKS)
+K8sSetup:
+  Platform: PlainK8s
+
+# No. of XL-Deploy master and worker pods to run.
+XldMasterCount: 3
+XldWorkerCount: 3
+
+## XL-Deploy image version
+## Ref: https://hub.docker.com/r/xebialabs/xl-deploy/tags
+ImageRepository: "xebialabs/xl-deploy"
+ImageTag: "10.0"
+
+## Specify a imagePullPolicy
+## Defaults to 'Always' if image tag is 'latest',set to 'IfNotPresent'
+ImagePullPolicy: "Always"
+
+## Secrets must be manually created in the namespace.
+# ImagePullSecret: xlDeploy
+
+## Install nginx subchart. If you have nginx already installed, set 'install' to 'false'.
+## If you have any other ingress controller installed, you can set the 'install' to 'false'.
+## Ref: https://github.com/bitnami/charts/blob/master/bitnami/nginx-ingress-controller/README.md
+nginx-ingress-controller:
+  install: true
```

Comment:
Add image pull secret option for Nginx-ingress controller

Reply:
updated values.yaml